packages/node-core/src/configure/migration-service/migration.ts

```diff
@@ -18,60 +61,115 @@ import {
 } from '../../utils/sync-helper';
 import {NodeConfig} from '../NodeConfig';
 
+type RemovedIndexes = Record<string, IndexesOptions[]>;
+
+const logger = getLogger('db-manager');
+
 export class Migration {
   private sequelizeModels: ModelStatic<any>[] = [];
-  private rawQueries: string[] = [];
+  private mainQueries: string[] = [];
   private readonly historical: boolean;
+  private extraQueries: string[] = [];
+  private foreignKeyMap: Map<string, Map<string, SmartTags>> = new Map<string, Map<string, SmartTags>>();
+  private useSubscription: boolean;
```

Comment:
This may have been a bug previously, but if this is based on a flag, do they get removed if you enable then disable?

Reply:
```suggestion
  private readonly useSubscription: boolean;
```